http-verbs-common/src/main/scala/uk/gov/hmrc/http/package.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc
+
+import java.net.URL
+
+import sttp.model.UriInterpolator
+
+package object http {
+  implicit class StringContextOps(sc: StringContext){
+    def url(args: Any*) : URL = UriInterpolator.interpolate(sc, args: _*).toJavaUri.toURL
```

Comment:
Providing our own string interpolation mechanism which uses sttp UriInterpolator. Ensure handle exceptions.

Reply:
It throws IllegalArgumentException as of now.